test/common/storage/base-test.js

```diff
@@ -22,12 +22,26 @@ var fs = require('fs'),
     File = require('../../../lib/pkgcloud/core/storage/file').File,
     mock = !!process.env.MOCK,
     pkgcloud = require('../../../lib/pkgcloud'),
-    fillerama = fs.readFileSync(helpers.fixturePath('fillerama.txt'), 'utf8');
+    fillerama = fs.readFileSync(helpers.fixturePath('fillerama.txt'), 'utf8'),
+    bigfileSize = fs.readFileSync(helpers.fixturePath('bigfile.raw')).length;
+
+// GitHub gets crabby with files over 100Mb, for good reason.  So we'll just generate
+// the really big (300Mb) test file.  The reason we need the file is that hock explodes
+// if you try to respond with a buffer that big, so we need to let hock respond with
+// a file. 
+var reallybigfileSize = 300000000;
+var fd =  fs.openSync(helpers.fixturePath('reallybigfile.raw'), 'a+');
+var buff = new Buffer(reallybigfileSize);
+for (var i = 0; i < buff.length; i++) {
+  buff[i] = i % 256;
+}
+fs.write(fd, buff, 0, buff.length, 0);
```

Comment:
I'm wondering if we even need to create the file on disk, versus just using a stream that we randomly send bits out of until we have sent enough.